lib/logmsg/logmsg.h

```diff
@@ -135,6 +135,7 @@ enum
   /* message field missing */
   LM_T_SYSLOG_MISSING_MESSAGE,
   LM_T_PREDEFINED_MAX,
+  LM_T_SYSLOG_RFC_3164_INVALID_PROGRAM,
```

Comment:
This should be moved one line above.

Reply:
Can we also insert a new line before `LM_T_PREDEFINED_MAX` macro so that we won't miss this next time?

Like it is done with `LM_V_PREDEFINED_MAX`.